index.html

```diff
@@ -15,176 +15,247 @@
-        </section>
-        <footer class="contact">
-            <h2>I'm always <br> interested in hearing about new projects, 
-                so if you'd like to chat please get in</h2>
-            <form class="ip-type" action="https://formspree.io/f/xzbokgee" method="post">
-                <input class="Name" type="text" placeholder="    Full Name" maxlength="30" required>
-                <input class="email" type="email" placeholder="   Email address" required>
-                <textarea name="msgt1" class="t1" cols="30" rows="10" maxlength="500" placeholder="  Enter text here" required></textarea> 
-                <button class="get" type="submit">Get in touch</button>
-            </form>
-            <hr class="asch">
-            <div class="ico">
-                <ul class="icons-last">
-                    <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
-                    <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
-                    <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
+            <div class="recta">
+                    <img src="img/diamond.png" alt="" class="circle">
+                    <h3>Skills</h3>
+                <ul class="rect-list">
+                    <li>Codekitt</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
                 </ul>
+            </div> 
+        </div>
+    </section>
+    <footer>
+        <div class="cont">
+            <div class="contact">
+                <p class="cont-dics">I'm always interested in hearing about new projects, 
+                    so if you'd like to chat please get in touch.</p>
+                <form class="ip-type" action="https://formspree.io/f/xzbokgee" method="post">
+                    <input type="text" class="input" placeholder="   Full Name" required>
+                    <input type="email" class="input" placeholder="   Email Address" required>
+                    <textarea name="msgt1" class="input-one" cols="30" rows="10" maxlength="500" placeholder="  Enter text here" required></textarea>
+                    <button type="button" class="cont-btn">Get in touch</button>
+                </form>
             </div>
-            <hr class="last">
-        </footer>
-    </main>
+        </div>
+        <div class="lin-la">
+            <hr class="lin">
+        </div>
+        <div class="last-icon">
+            <a href="#"><i class="fa-brands fa-github"></i></a>
+            <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+            <a href="#"><i class="fa-brands fa-angellist"></i></a>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            <a href="#"><i class="fa-solid fa-m"></i></a>
+        </div>
```

Comment:
- [ ] Kindly decrease the `font-size` of icons to make similar to the Figma design
### your design
![his](https://user-images.githubusercontent.com/81727988/166094221-c8fc8b06-dc31-4847-9294-2ff15f1dc9d3.png)
### Figma design
![icons](https://user-images.githubusercontent.com/81727988/166094233-106097bb-037c-4583-b104-7097a39bef35.png)